.github/workflows/label-progress.yml

```
@@ -1,14 +1,84 @@
-name: Label when moved to In Progress
+name: Auto Label on Project Status Change
 
 on:
-  project_card:
-    types: [moved]
+  project_v2_item:
```

actionlint (1.7.10)

[error] unknown Webhook event "project_v2_item". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)